internal/controller/fusionaccess_controller.go

```diff
@@ -309,12 +309,18 @@ func (r *FusionAccessReconciler) Reconcile(
 	if err != nil {
 		return ctrl.Result{}, err
 	}
-	// Load and install manifests from ibm
+	// Load and install manifests from ibm (from the repo)
 	install_path, err := utils.GetInstallPath(string(fusionaccess.Spec.IbmCnsaVersion))
 	if err != nil {
 		return ctrl.Result{}, err
 	}
 
+	// Load and install manifests from external url if defined
+	if fusionaccess.Spec.ExternalManifestURL != "" {
```

Comment:
Can we do some kind of validation for this URL? maybe allowed domains or patterns?